lib/reporters/json.js

```diff
@@ -15,6 +17,10 @@ exports = module.exports = JSONReporter;
 /**
  * Initialize a new `JSON` reporter.
  *
+ * @public
+ * @class JSON
+ * @memberof Mocha.reporters
+ * @extends Mocha.reporters.Base
  * @api public
```

Comment:
You are welcome to remove `@api public` and `@api private` from everywhere if you wish.